newrelic/hooks/datastore_aredis.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2010 New Relic, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import re
+
+from newrelic.common.object_wrapper import wrap_function_wrapper
+from newrelic.hooks import datastore_redis
+
+
+def instrument_aredis_client(module):
```

Comment:
I tried running a test application that uses these instrumentation functions and I'm not seeing any redis calls being captured. Am I validating this code the same way that you did locally?

Config file:
```ini
[newrelic]
app_name = aredis
log_file = stderr
log_level = debug
distributed_tracing.enabled = true

[import-hook:aredis.client]
enabled = true
execute = newrelic.hooks.datastore_aredis:instrument_aredis_client

[import-hook:aredis.connection]
enabled = true
execute = newrelic.hooks.datastore_aredis:instrument_aredis_connection
```

Test Program (app.py):
```python
import asyncio

from aredis import StrictRedis

import newrelic.agent


@newrelic.agent.background_task()
async def example():
    client = StrictRedis(host="127.0.0.1", port=6379, db=0)
    await client.flushdb()
    breakpoint()
    await client.set("foo", 1)
    assert await client.exists("foo") is True
    await client.incr("foo", 100)

    assert int(await client.get("foo")) == 101
    await client.expire("foo", 1)
    await asyncio.sleep(0.1)
    await client.ttl("foo")
    await asyncio.sleep(1)
    assert not await client.exists("foo")


loop = asyncio.get_event_loop()
loop.run_until_complete(example())
```

Command (assumes NEW_RELIC_LICENSE_KEY env variable set):
`NEW_RELIC_CONFIG_FILE=newrelic.ini newrelic-admin run-program python app.py`